docs/samples/Microsoft.ML.Samples/Dynamic/MatrixFactorization.cs

```diff
@@ -10,23 +10,23 @@ public class MatrixFactorizationExample
         // The variable _synthesizedMatrixFirstRowIndex indicates the integer that would be mapped to the first row index. If user data uses
         // 0-based indices for rows, _synthesizedMatrixFirstRowIndex can be set to 0. Similarly, for 1-based indices, _synthesizedMatrixFirstRowIndex
         // could be 1.
-        const int _synthesizedMatrixFirstColumnIndex = 1;
```

Comment:
Let's keep these with the underscore (`_`), but mark them private.

Reply:
Ok I didn't think any names should have an underscore regardless of access

Reply:
The convention we follow is that private variables start with underscores and public variables start with caps.